### ISSUE TITLE
Training and evaluation configs for BLIP2 on the NoCaps dataset

### ISSUE THREAD

**Issue by albert-cwkuo** [salesforce/LAVIS#190]
Great work and thanks for releasing the code and checkpoints for BLIP2.

One thing we noticed missing is the training and evaluation config on the NoCaps dataset. We looked into `LAVIS/run_scripts/blip2/` and `LAVIS/lavis/projects/blip2/` but only found the configs for COCO and other tasks. It would be great if you could release these configs, too.

**Comment by dxli94** [comment]
Nocap is for zero-shot evaluation. No training is conducted.

Evaluation is done via official server.

**Comment by albert-cwkuo** [comment]
Yeah I understand that. But in the paper, it's mentioned that the model is fine-tuned on COCO. I wonder if you use different training hyperparameters than coco training?

Also, what are the config and hyperparameters for generating NoCaps captions? Do you use the same for COCO?
